README.md

```diff
@@ -1,6 +1,130 @@
+
+1. VM Live Migration: During VM live migration, both source and target pods are required.
+Having both pods simultaneously affects quotas, potentially leading to migration issues and upgrade failures.
+2. Billing Considerations: Service providers typically bill VMs similarly to platforms 
+like OpenStack, based on RAM size and CPU threads. However, KubeVirt's pods must account 
+for infrastructure overhead, which complicates resource quota management.
+
+
+As a result, deploying KubeVirt in a multi-tenant cluster is highly challenging. 
+It requires significant resource allocation to handle live migrations and manage VM infrastructure 
+overhead. This also couples users to the VM implementation, making them dependent on infrastructure 
+overhead that can change unexpectedly.
+
+
+## Overcoming Resource Quota Challenges with ApplicationAwareResourceQuota
+
+To address these challenges, the AAQ Operator use a different approach compared to the 
+native ResourceQuota. To avoid API server overhead, the AAQ Operator introduces an alternative 
+quota implementation called ApplicationAwareResourceQuota.
+The ApplicationAwareResourceQuota is API compatible with the native ResourceQuota, meaning it shares the same [specification and status definitions](https://github.com/kubevirt/application-aware-quota-api/blob/ac6757c09496bfab205d499a41daa6ed8815eab6/pkg/apis/core/v1alpha1/types.go#L46).
+Please see `manifests/examples/arq-example.yaml` for an example manifest.
+
+This new quota is controller-based rather than admission-based. Here’s how it works:
+
+1. Scheduling Gates: Scheduling gates are added to pods within namespaces where the Quotas are enforced using a MutatingWebhookConfiguration Object.
+2. Controller Evaluation: Once a pod is created, a controller evaluates whether there is sufficient room for the pod in the namespace.
+3. Quota Update: If there is enough room, the scheduling gate is removed from the pod, and the quota used status is updated accordingly.
+
+
+Additionally, the AAQ Operator allows application owners to define their own policies for how resources 
+should be counted for pods associated with their applications. This enables application owners to hide infrastructure overhead, 
+which is essentially an implementation detail.
+
+## Multi-Namespace Quota 
+
+The AAQ Operator includes a Multi-Namespace Quota called ApplicationAwareClusterResourceQuota, 
+similar to the ClusterResourceQuota available in OpenShift distributions. 
+This quota is API compatible with ClusterResourceQuota, meaning it shares the same [specification and status definitions](https://github.com/kubevirt/application-aware-quota-api/blob/ac6757c09496bfab205d499a41daa6ed8815eab6/pkg/apis/core/v1alpha1/types.go#L257). 
+
+For more information, please see this [link](https://docs.openshift.com/container-platform/4.15/applications/quotas/quotas-setting-across-multiple-projects.html).
```

Comment:
I would remove this reference. We can maybe say we're inspired by it, but not reference it as a source of documentation IMO

Reply:
Done